build.gradle

```diff
@@ -24,36 +25,17 @@ archivesBaseName = 'thrift-gradle-plugin'
 version = '0.4.2'
 description = 'A gradle plugin for compiling Thrift IDL files using thrift compiler'
 
-ext {
-	title = 'Gradle Thrift Plugin'
-	organizationId = 'org.jruyi'
-	organizationName = 'JRuyi.org'
-	organizationUrl = 'http://www.jruyi.org/'
-
-	websiteUrl = 'https://github.com/jruyi/thrift-gradle-plugin'
-	issueTrackerUrl = 'https://github.com/jruyi/thrift-gradle-plugin/issues'
-	vcsUrl = 'https://github.com/jruyi/thrift-gradle-plugin.git'
-
-	licenseName = 'Apache License, Version 2.0'
-	licenseUrl = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
-
-	createdBy = "${System.getProperty('java.version')} (${System.getProperty('java.vm.vendor')})"
-}
-
-configurations.all {
-	transitive = false
+repositories {
+	mavenCentral()
 }
 
 dependencies {
-	compile gradleApi()
-	compile localGroovy()
-}
-
-processResources {
-
-	from(['LICENSE', 'NOTICE']) {
-		into 'META-INF'
-	}
+	implementation gradleApi()
+	implementation localGroovy()
+	testImplementation "org.assertj:assertj-core:3.24.2"
+	testImplementation "org.junit.jupiter:junit-jupiter:5.9.2"
+	testImplementation "org.mockito:mockito-junit-jupiter:5.1.1"
+	testImplementation "org.mockito.kotlin:mockito-kotlin:4.1.0"
```

Comment:
nit: should we use the latest version of these testing libs?

* junit-jupiter: 5.9.3
* mockito-junit-jupiter: 5.4.0
* mockito-kotlin: 5.0.0